docs/src/content/docs/es/challenges/angular/3-directive-enhancement.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 🟠 Mejorar Directiva
+description: El desafío 3 se trata de una directive incorporada.
+author: thomas-laforge
+challengeNumber: 3
+command: angular-ngfor-enhancement
+blogLink: https://medium.com/@thomas.laforge/ngfor-enhancement-716b44656a6c
+sidebar:
+  order: 101
+---
+
+:::note
+Este desafío puede parecer obsoleto ahora que disponemos del nuevo control de flujo y el bloque vacío dentro de los bloques `@for`. Sin embargo, las **directivas estructuradas** no serán borradas en ningún momento, de esta manera usted tendrá la oportunidad de aprender bastante de este desafió.
+:::
+
+## Información
+
+Las Directivas en Angular Framework son una herramienta muy poderosa. Usted puede aplicar los principios de DRY para aplicar una lógica compartida dentro de una directiva y aplicarla a cualquier componente.
+
+Pero el verdadero poder esta en mejorar una directiva ya existente que además **no te pertenece**.
+
+## Declaración
+
+En este desafío, queremos mostrar una lista de personas. Si la lista esta vacía, usted deberá mostrar _" the list is empty !! "_. (La lista esta vacía)
+
+:::note
+_(Notas del traductor)_
+
+En caso que hayan test automáticos para validar este desafío, he preferido dejar la frase en ingles, en vez de utilizar una traducción directa al español.
+:::
```

Comment:
Why did you added this ?

Reply:
ok, sounds fair. I can edit it to reflect it in a more generic scenario rather than a note from the translator.